addons/account_edi/models/account_edi_document.py

```diff
@@ -127,7 +127,10 @@ def _postprocess_post_edi_results(documents, edi_result):
                     if not old_attachment.res_model or not old_attachment.res_id:
                         attachments_to_unlink |= old_attachment
                 if move_result.get('success') is True:
-                    document.state = 'sent'
+                    document.write({'state': 'sent',
+                                    'error': '',
+                                    'blocking_level': ''})
```

Comment:
```suggestion
                    document.write({
                        'state': 'sent',
                        'error': False,
                        'blocking_level': False,
                    })
```

Keep the same style and value as in `_postprocess_cancel_edi_results` (copy paste is bad :grimacing:)
Also, why is the `attachment_id` set to `False` when cancelling?